<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageInfoDialog</class>
 <widget class="QDialog" name="ImageInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabProperty">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <widget class="QLabel" name="pr_sizeLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Size: </string>
       </property>
      </widget>
      <widget class="QLabel" name="pr_bytesLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Length: </string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabDisplaySettings">
      <attribute name="title">
       <string>Display Settings</string>
      </attribute>
      <widget class="QLabel" name="ds_lb_0">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>66</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>onColor</string>
       </property>
      </widget>
      <widget class="QLabel" name="ds_lb_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>66</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>offColor</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="ds_cbxShowGrid">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Grid</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabNewImage">
      <attribute name="title">
       <string>New Image</string>
      </attribute>
      <widget class="QSpinBox" name="ni_spnWidth">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>11</y>
         <width>71</width>
         <height>26</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="ni_spnHeight">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>41</y>
         <width>71</width>
         <height>26</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
      <widget class="IndicatorButton" name="ni_chkAspectRatio">
       <property name="geometry">
        <rect>
         <x>97</x>
         <y>24</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="uires.qrc">
         <normaloff>:/link</normaloff>:/link</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabAdjust">
      <attribute name="title">
       <string>Adjust</string>
      </attribute>
      <widget class="QSpinBox" name="ad_spnWidth">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>11</y>
         <width>71</width>
         <height>26</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="ad_spnHeight">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>41</y>
         <width>71</width>
         <height>26</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>99</width>
         <height>115</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
	border: 1px solid gray;
	margin-top: 2ex;
}
QGroupBox::title {
	subcontrol-origin: margin;
	margin-left: 2px;
}</string>
       </property>
       <property name="title">
        <string>Alignment</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="2">
         <widget class="IndicatorButton" name="ad_pbAlignTR">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="uires.qrc">
            <normaloff>:/arrow-tr</normaloff>:/arrow-tr</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="IndicatorButton" name="ad_pbAlignL">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="uires.qrc">
            <normaloff>:/arrow-left</normaloff>:/arrow-left</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="IndicatorButton" name="ad_pbAlignTL">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="uires.qrc">
            <normaloff>:/arrow-tl</normaloff>:/arrow-tl</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="IndicatorButton" name="ad_pbAlignT">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="uires.qrc">
            <normaloff>:/arrow-up</normaloff>:/arrow-up</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="IndicatorButton" name="ad_pbAlignC">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="uires.qrc">
            <normaloff>:/position</normaloff>:/position</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="IndicatorButton" name="ad_pbAlignBL">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="uires.qrc">
            <normaloff>:/arrow-bl</normaloff>:/arrow-bl</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="IndicatorButton" name="ad_pbAlignR">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="uires.qrc">
            <normaloff>:/arrow-right</normaloff>:/arrow-right</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="IndicatorButton" name="ad_pbAlignB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="uires.qrc">
            <normaloff>:/arrow-down</normaloff>:/arrow-down</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="IndicatorButton" name="ad_pbAlignBR">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="uires.qrc">
            <normaloff>:/arrow-br</normaloff>:/arrow-br</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="IndicatorButton" name="ad_chkAspectRatio">
       <property name="geometry">
        <rect>
         <x>96</x>
         <y>24</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="uires.qrc">
         <normaloff>:/link</normaloff>:/link</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbExit">
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IndicatorButton</class>
   <extends>QPushButton</extends>
   <header>indicatorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="uires.qrc"/>
 </resources>
 <connections/>
</ui>
